src/main/java/me/fallenbreath/tweakermore/config/MalilibStuffsInitializer.java

```diff
@@ -32,7 +37,9 @@ public static void init()
 	{
 		InitializationHandler.getInstance().registerInitializationHandler(() -> {
 			ConfigManager.getInstance().registerConfigHandler(TweakerMoreMod.MOD_ID, TweakerMoreConfigStorage.getInstance());
-
+			//#if MC >= 12101
```

Comment:
what is this for? new api?

Reply:
That's the Config Switcher Registry.

Reply:
The Addition of correct registration makes it so that you can control how the text is displayed for your Mod Name

Reply:
This is one of those Post-ReRwite functions that I was referring to.

Reply:
![2024-12-30_13 13 19](https://github.com/user-attachments/assets/2e81e135-dc8a-4524-9804-e3e76c43a307)
![2024-12-30_13 13 27](https://github.com/user-attachments/assets/4582a859-a1b4-47e7-9827-4a22f962fd77)

It does this.